src/android/toga_android/layout.py

```diff
@@ -27,7 +27,8 @@ def onLayout(self, changed: bool, left: int, top: int, right: int, bottom: int)
         # print("LAYOUT: There are %d children" % count)
         for i in range(0, count):
             child = self.getChildAt(i)
-            # print("    child: %s" % child, child.getMeasuredHeight(), child.getMeasuredWidth(), child.getWidth(), child.getHeight())
+            # print("    child: %s" % child, child.getMeasuredHeight(), child.getMeasuredWidth(),
```

Comment:
Do we actually want this comment?

Reply:
Probably not. I've adjusted it in a follow-up!